docs/framework/data/adonet/updating-data-sources-with-dataadapters.md

```diff
@@ -71,7 +71,7 @@ adapter.Update(table.Select(Nothing, Nothing, _
   DataViewRowState.ModifiedCurrent))  
   
 ' Finally, process inserts.  
-dataAdapater.Update(table.Select(Nothing, Nothing, _  
+dataAdapter.Update(table.Select(Nothing, Nothing, _  
```

Comment:
I'd just change this to match the C# version

```suggestion
adapter.Update(table.Select(Nothing, Nothing, _  
```

Reply:
Arg, the branch merge messed up the suggestion

Reply:
I might have done something wrong when resolving the conflict. Did a new suggestion.